docs/atsd_install.md

```diff
@@ -0,0 +1,162 @@
+# Installing Axibase Time-Series Database (ATSD) as the backend for cAdvisor 
+
+[Axibase Time Series Database](http://axibase.com/products/axibase-time-series-database/) can collect Docker metrics through cAdvisor for long-term retention, analytics and visualization. A single ATSD instance can collect metrics from many Docker hosts and cAdvisors.
+
+In the standard setup cAdvisor monitors the Docker Host, ATSD container and itself. All statistics are sent over TCP protocol to ATSD. Other local containers will be automatically monitored by cAdvisor and their statistics will be sent to ATSD.
+
+Remote Docker hosts can be easily monitored with multiple cAdvisor instances sending data to a centralized ATSD server. ATSD will store metrics from local and remote Docker hosts for consolidated monitoring and analytics. This type of setup will allow for centralized capacity planning and performance monitoring.
+
+![Distributed Docker Infrastructure](http://axibase.com/wp-content/uploads/2015/01/docker_distributed.png)
+
+[Learn more about cAdvisor](http://axibase.com/products/axibase-time-series-database/writing-data/docker-cadvisor/)
+
+#### Quick Start: Running cAdvisor in a Docker Container with ATSD as the storage driver
+
+Start ATSD in new container:
+
+```
+sudo docker run \
+  -d \
+  -p 5022:22 \
+  -p 8088:8088 \
+  -p 8081:8081 \
+  -p 8443:8443 \
+  -p 8082:8082/udp \
+  -e AXIBASE_USER_PASSWORD=secret-pwd \
+  -e ATSD_USER_NAME=user \
+  -e ATSD_USER_PASSWORD=secret-pwd \
+  -h atsd \
+  --name=atsd \
+  axibase/atsd
+```
+
+Launch cAdvisor container with ATSD storage driver:
+
+With keys:
+
+```
+sudo docker run \
+  --volume=/:/rootfs:ro \
+  --volume=/var/run:/var/run:rw \
+  --volume=/sys:/sys:ro \
+  --volume=/var/lib/docker/:/var/lib/docker:ro \
+  --publish=8080:8080 \
+  --detach=true \
+  --name=cadvisor \
+  --link atsd:atsd \
+  axibase/cadvisor:latest \
+  --storage_driver=atsd \
+  --storage_driver_user=user \
+  --storage_driver_password=secret-pwd \
+  --storage_driver_atsd_url=http://atsd:8088 \
+  --storage_driver_atsd_write_host=atsd:8081 \
+  --storage_driver_atsd_write_protocol=tcp \
+  --storage_driver_atsd_docker_host="`hostname`" \
+  --storage_driver_atsd_property_interval=15s \
+  --housekeeping_interval=15s \
```

Comment:
We think that 15 second collection interval is more appropriate for a long-term retention. Since the metrics will be sent to ATSD at this frequency it does not make sense to collect them more frequently than necessary.